polaris-react/build/esm/styles.css

```diff
@@ -13043,39 +13118,28 @@ li:first-of-type > .Polaris-Listbox-TextOption {
   justify-content: flex-end;
   text-align: right;
 }
-@media (max-width: 64.9975em) {
-  .Polaris-Page-Header--longTitle .Polaris-Page-Header__AdditionalMetaData {
-    margin-left: 0;
-  }
-  .Polaris-Page-Header--longTitle .Polaris-Page-Header__Row {
-    display: grid;
-    gap: var(--p-space-200) var(--p-space-400);
-    grid-template-columns: auto 1fr;
-    grid-template-areas: 'breadcrumbs actions' 'title title';
-  }
-  .Polaris-Page-Header--longTitle
-    .Polaris-Page-Header__Row
-    + .Polaris-Page-Header__Row {
-    gap: 0;
-  }
+.Polaris-Page-Header--longTitle.Polaris-Page-Header--lgDown
+  .Polaris-Page-Header__AdditionalMetaData,
+.Polaris-Page-Header--mediumTitle.Polaris-Page-Header--mdDown:not(.Polaris-Page-Header--noBreadcrumbs)
+  .Polaris-Page-Header__AdditionalMetaData {
+  margin-left: 0;
 }
-@media (max-width: 47.9975em) {
-  .Polaris-Page-Header--mediumTitle:not(.Polaris-Page-Header--noBreadcrumbs)
-    .Polaris-Page-Header__AdditionalMetaData {
-    margin-left: 0;
-  }
-  .Polaris-Page-Header--mediumTitle:not(.Polaris-Page-Header--noBreadcrumbs)
-    .Polaris-Page-Header__Row {
-    display: grid;
-    gap: var(--p-space-200) var(--p-space-400);
-    grid-template-columns: auto 1fr;
-    grid-template-areas: 'breadcrumbs actions' 'title title';
-  }
-  .Polaris-Page-Header--mediumTitle:not(.Polaris-Page-Header--noBreadcrumbs)
-    .Polaris-Page-Header__Row
-    + .Polaris-Page-Header__Row {
-    gap: 0;
-  }
+.Polaris-Page-Header--longTitle.Polaris-Page-Header--lgDown
+  .Polaris-Page-Header__Row,
+.Polaris-Page-Header--mediumTitle.Polaris-Page-Header--mdDown:not(.Polaris-Page-Header--noBreadcrumbs)
+  .Polaris-Page-Header__Row {
+  display: grid;
+  gap: var(--p-space-200) var(--p-space-400);
+  grid-template-columns: auto 1fr;
+  grid-template-areas: 'breadcrumbs actions' 'title title';
+}
+.Polaris-Page-Header--longTitle.Polaris-Page-Header--lgDown
+  .Polaris-Page-Header__Row
+  + .Polaris-Page-Header__Row,
+.Polaris-Page-Header--mediumTitle.Polaris-Page-Header--mdDown:not(.Polaris-Page-Header--noBreadcrumbs)
+  .Polaris-Page-Header__Row
+  + .Polaris-Page-Header__Row {
+  gap: 0;
```

Comment:
The `Page.Header` component was [relying on mixins to avoid duplicating styles](https://github.com/Shopify/polaris/blob/90a3cb1fb3fb8ca26807e7b17b11bd2fe4c0c7a1/polaris-react/src/components/Page/components/Header/Header.module.scss#L139-L149).

Since mixins in `postcss-mixins` are all global, we opted to refactor these selectors so it relied on a `.lgDown` class instead of a mixin-in-breakpoint approach. The code diff looks big, but really the changes are minimal.